src/Microsoft.AspNetCore.Cors/Infrastructure/CorsPolicy.cs

```diff
@@ -141,5 +154,10 @@ public override string ToString()
             builder.Append("}");
             return builder.ToString();
         }
+
+        private bool DefaultIsOriginAllowed(string origin)
+        {
+            return Origins.Contains(origin);
```

Comment:
Changing to `OridinalIgnoreCase` also breaks the following test:

`Microsoft.AspNetCore.Cors.Infrastructure.CorsServiceTests.TryValidateOrigin_DoesCaseSensitiveComparison`

So for now I am going to set it to `Ordinal`. Unless you are happy for me to change this test case?